src/index.html

```diff
@@ -1,14 +1,31 @@
 <!doctype html>
-<html lang="en">
+<html lang="en ru">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div>
+        <a href="#home" class="logo"><img src="images/logo.png" alt=" "></a>
```

Comment:
go through [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md) and fix your codestyle